src/main/java/org/example/csv2tex/placeholders/schoolspecific/ErfurtSchoolTablePlaceholderReplacer.java

```diff
@@ -151,21 +174,41 @@ String makeCompetencyEntriesMS(List<SchoolCompetencyData> competencyList) {
 
         for (SchoolCompetencyData schoolCompetencyData : competencyList) {
             StringBuilder competency = new StringBuilder();
-            appendBold(competency, schoolCompetencyData.schoolCompetency);
+            if (!schoolCompetencyData.schoolCompetency.isEmpty()) {
```

Comment:
These new conditionals deserve a test case each, imo (line coverage does not equal branch coverage).